_posts/2016-05-26-Rust-1.9.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: "Announcing Rust 1.9"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.9. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.9][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.9][notes] on GitHub.
+About 1400 patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-190-2016-05-26
+
+### What's in 1.9 stable
+
+Rust 1.9 contains some exciting new things!
+
+First up, a new attribute for library authors: `#[deprecated]`. Authors of
+libraries can now tag items with this attribute, and users of their crate
+will see a warning when it is used. This new feature was defined in [RFC 1270].
+
+[RFC 1270]: https://github.com/rust-lang/rfcs/blob/master/text/1270-deprecation.md
+
+We now publishes std binaries for the `mips-unknown-linux-musl`,
+`mipsel-unknown-linux-musl`, and `i586-pc-windows-msvc` targets.
+
+[The time complexity of comparing variables for equivalence][compare] during
+type unification is reduced from O(n!) to O(n). What does this mean? Certain
+kinds of code compiles much, much faster.
+
+[compare]: https://github.com/rust-lang/rust/pull/32062
+
+Strings are a very common type, and implement a lot of different traits.
+Specifically, when converting between `&str` and `String`, you have many
+options. Rustaceans used these two the most:
+
+```rust
+let s = "some string".to_string();
+let s = "some string".to_owned();
+```
+
+The former is the official convention, but it was discovered that it's not
+as fast as the second method. For a long time, it was not possible to improve
+this situation, however, a new language feature has landed on nightly and made
```

Comment:
maybe worth naming the feature here? "a new language featured called specialization"?